sdk/rust/src/token/custom_fees.rs

```diff
@@ -24,6 +24,7 @@ use hedera_proto::services;
 use crate::{
```

Comment:
Just as a note - we need to rewrite the custom fee types here to match what they are in Swift/Java/etc.